src/Redux/Country/countrySlice.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Linters are there to help you achieve standard best practices, so it is not the best to disable any of them. Kindly make sure you remove this line. :+1: